spiped/spiped.1

```diff
@@ -153,5 +153,13 @@ Print version number.
 .TP
 .B \-1
 Exit after a connection closes.  Used for internal testing.
+.SH SIGNALS
+.B SIGTERM
```

Comment:
hmm, for what it's worth, tarsnap.1 uses these commands:

```
.Sh SIGNALS
.Nm
provides special treatment of the following signals:
.Bl -tag -width "SIGUSR2"
.It SIGUSR1 & SIGINFO
On receipt of the SIGUSR1 signal or (on platforms where it exists) the
SIGINFO signal,
```

which results in it being formatted as:

```
SIGNALS
     tarsnap provides special treatment of the following signals:

     SIGUSR1 & SIGINFO
              On receipt of the SIGUSR1 signal or (on platforms where it
              exists) the SIGINFO signal,
```

I personally like that formatting a bit more than the proposed change, which doesn't indent the explanation paragraph relative to the `SIGTERM`:

```
SIGNALS
       SIGTERM When spiped receives the SIGTERM signal it will stop  accepting
       new connections and exit once there are no active connections left.
```